execute/executetest/table.go

```diff
@@ -475,11 +476,58 @@ func (b SortedTables) Swap(i int, j int) {
 	b[i], b[j] = b[j], b[i]
 }
 
-// NormalizeTables ensures that each table is normalized
+// NormalizeTables ensures that each table is normalized and that tables and columns are sorted in
+// alphabetical order for consistent testing
 func NormalizeTables(bs []*Table) {
 	for _, b := range bs {
 		b.Key()
 	}
+	sortByGroupKey(bs)
+}
+
+func sortByGroupKey(tables []*Table) {
+	sort.Sort(SortedTables(tables))
+	for _, table := range tables {
+		sortColumns(table)
+	}
+}
+
+func sortColumns(table *Table) {
+	// index reference to make sure that sort is consistent for metadata and data
+	indexes := createReferenceArray(table)
+
+	columnsCopy := make([]flux.ColMeta, len(table.ColMeta))
+	for i, value := range indexes {
+		columnsCopy[i] = table.ColMeta[value]
+	}
+	table.ColMeta = columnsCopy
+
+	// don't create a new table.Data if there is no Data
+	if len(table.Data) == 0 {
+		return
+	}
+	valuesCopy := make([][]interface{}, len(table.Data))
```

Comment:
Do we need to make a full `valuesCopy`? Can we get away with just copying the row as we need it, since we're not re-ordering the rows themselves?

Reply:
We still need the outer array to house the interface array.